core/server/i18n.js

```diff
@@ -50,9 +69,9 @@ I18n = {
      * @param {string} msgPath Path with in the JSON language file to desired string (ie: "errors.init.jsNotBuilt")
      * @returns {string}
      */
-    findString: function findString(msgPath, opts) {
+    findString: function findString(msgPath, defaultString, opts) {
```

Comment:
https://github.com/TryGhost/Ghost/blob/master/core/server/i18n.js#L88

`doesTranslationKeyExist` uses `msgPath, opts`.

Reply:
Fixed yesterday. Added missing empty parameter for `findString` in `I18n.doesTranslationKeyExist`.

Reply:
Can we please combine `defaultString` and `opts`? If we keep `defaultString`, we could threat `defaultString` as option.

> var translation = I18n.findString(msgPath, '', {log: false});

^ This is hard to read.

Reply:
Done.